src/components/Lobby/Lobby.js

```diff
@@ -29,10 +38,10 @@ const Lobby = () => {
   }, [JSON.stringify(errors), toast.error]);
 
   /** Tell server to begin game */
-  const roomFull = () => {
+  const roomFull = (gameId, gameConfig) => {
     // the game requires two players to begin
     if (playerList.length >= 2) {
-      socket.emit('hostRoomFull', roomId);
+      socket.emit('hostRoomFull', roomId, gameId, gameConfig);
```

Comment:
See my comment in backend code, try to only refer to gameId in backend and roomId in frontend, they are the same value. This separation in naming lets us swap out what we use for roomId, if we ever need to do that.